app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Recipe for <%= @recipe.name %></h1>
+<h3><%= @recipe.genre %> Recipe, Complexity: <%= @recipe.complexity %></h3>
+<h3>Ingredients Needed:</h3>
+<ul>
+  <% @recipe.ingredients.each do |ingredient| %>
+  <li><%= ingredient.name %></li>
+  <% end %>
+</ul>
+<h4>Total Recipe Cost: <%= @total_cost %></h4>
+<%= form_with url: "/recipes/update", method: :get, data: { turbo: false } do |form| %>
```

Comment:
See comment in the controller action first! Some additional thoughts:
- As mentioned, I think the most RESTful convention here would be to do a `POST` to /recipes/:recipe_id/ingredients`. That way you have the recipe ID in the path. And you're using a POST to indicate that you're creating a joins record, and you could direct this post to a RecipeIngredientsController